test/e2e-helm/e2e_helm_suite_test.go

```diff
@@ -56,7 +54,6 @@ var _ = BeforeSuite(func(done Done) {
 	tc, err = testutils.NewTestContext("GO111MODULE=on")
 	Expect(err).NotTo(HaveOccurred())
 	Expect(tc.Prepare()).To(Succeed())
-	projectName = filepath.Base(tc.Dir)
```

Comment:
Nit: IMO, we should keep projectName as a variable so that it encourages reuse if we add test code later that needs it.

Reply:
I do not see the point to keep it since the project name is not used at all. So, we are keeping an unused code which for me shows a bad practice. Also, we should not use the `filepath.Base(tc.Dir)` as the projectName since users are able to change this value in the PROJECT file.

Reply:
It is no longer more the case. We moved that for the TestContext.